test/base/babel-tests.js

```diff
@@ -663,11 +670,32 @@ test("enumerability - workaround", () => {
     expect("a" in a).toBe(true)
     expect(a.hasOwnProperty("a")).toBe(true)
     expect(a.hasOwnProperty("a2")).toBe(true)
-    expect(a.hasOwnProperty("b")).toBe(false) // true would also be ok-ish. see: #1398
+    expect(a.hasOwnProperty("b")).toBe(false) // ideally false, true would also be ok-ish. see: #1398
     expect(a.hasOwnProperty("m")).toBe(false)
     expect(a.hasOwnProperty("m2")).toBe(true)
 })
 
+test.skip("Babel bug", () => {
+    // See: https://github.com/babel/babel/issues/8760
+    class Todo1 {
+        id = 1
+    }
+
+    function dec(el) {
+        return el
+    }
+    class Todo2 {
+        id = 1
+        @dec x
+    }
+
+    const t1 = new Todo1()
+    expect(Object.getOwnPropertyDescriptor(t1, "id").enumerable).toBe(true)
+
+    const t2 = new Todo2()
+    expect(Object.getOwnPropertyDescriptor(t2, "id").enumerable).toBe(true) // fails
```

Comment:
I believe that this is fixed in Babel 7.1.3

Reply:
Confirmed